src/windows/views/properties_tableview.py

```diff
@@ -181,7 +181,7 @@ def mouseMoveEvent(self, event):
             except Exception:
                 # If item is deleted during this drag... an exception can occur
                 # Just ignore, since this is harmless
-                return
+                log.warning('Failed to access data on selected label widget')
```

Comment:
```suggestion
                log.debug('Failed to access data on selected label widget')
```
Although, since the comment gives a perfectly good reason for this happening normally, logging it at all doesn't really seem necessary. There's a reason Codacy allows its notations to be Ignored on either an individual or repo-wide basis, some "issues" really aren't.